src/app/components/header/header.component.css

```diff
@@ -0,0 +1,30 @@
+nav{
+    letter-spacing: 2.5px;
```

Comment:
use integer values for pixels - so 2px or 3px - not decimal values - as this is a bad practice which leads to inconsistent results between browsers

Reply:
okay, noted and changed to 2